### DISCUSSION TITLE
Is there any public API information for Dokku Pro?

### DISCUSSION THREAD

**Discussion by voycey** [dokku/dokku#5436] [Q&A] [answered]
We are considering using Dokku Pro for a project but I would like to see the API endpoints for the Dokku Pro API before pulling the trigger and buying the code, is this available anywhere? I can only see instructions to point to the swagger UI on the pro version (which cant be accessed without purchasing)?

**Comment by josegonzalez** [comment]
Hey @voycey!

I don't have anything public atm, but I should be able to publish the spec/swagger ui somewhere for this purpose. It would point to a non-existent install of course, but would that be enough?

**Comment by voycey** [comment]
More than enough mate thanks! I just need to model what we want against the endpoints before pulling the pin on pro!

Hope you are well :)

**Comment by josegonzalez** [comment, marked as answer]
There is now a link to the [openapi.yml spec](https://pro.dokku.com/docs/features/api/openapi.yml) on the [general documentation site](https://pro.dokku.com/docs/features/api/). This will be updated as I push newer versions of the api, and thus might be slightly more advanced than what is released at a point in time.

**Comment by voycey** [comment]
Thanks @josegonzalez 🙏
